driver-sync/src/main/com/mongodb/client/MongoClient.java

```diff
@@ -101,7 +102,6 @@
      * @return the list databases iterable interface
      */
     ListDatabasesIterable<Document> listDatabases();
-
```

Comment:
This is a judgement call, but generally try to avoid unrelated whitespace changes.  As the commits get bigger, they can distract from the important parts.

Whitespace and other formatting issues can always be done as a separate commit, and don't required a Jira issue.

Reply:
I think that was by accident. I'll undo that change.